BlueDBOnDisk/src/test/java/org/bluedb/disk/collection/CollectionMetaDataTest.java

```diff
@@ -74,13 +74,13 @@ public void test_getAndAddToSerializedClassList() throws Exception {
 		Class<? extends Serializable>[] testValue1PlusDefaults = concatenate(defaultClasses, testValue1);
 		Class<? extends Serializable>[] testValueBothPlusDefaults =  concatenate(defaultClasses, testValueBoth);
 
-		Class<? extends Serializable>[] afterAdding1 = metaData.getAndAddToSerializedClassList(TestValue.class);
+		Class<? extends Serializable>[] afterAdding1 = metaData.getAndAddToSerializedClassList(TestValue.class, Arrays.asList());

```

Comment:
Its kind of annoying that we have to pass something now even if its just an empty list. It might be worth making a method that doesn't take in a list that calls the existing method and passes in the empty list.

Reply:
I created that method and then deleted it because it was only used for tests anyway.